examples/command_bot.rs

```diff
@@ -98,6 +82,14 @@ async fn login_and_sync(
 
     println!("logged in as {}", username);
 
+    // initial sync to set up state and so our bot doesn't respond to old messages
+    client.sync(SyncSettings::default()).await.unwrap();
```

Comment:
The `sync_forever()` method doesn't use the sync_token from the base client automatically, we either need to change that or you'll need to pass it in the `SyncSettings` for the `sync_forever()` method.